akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -58,9 +58,9 @@ abstract class JournalSpec(config: Config) extends PluginSpec(config) {
   def writeMessages(fromSnr: Int, toSnr: Int, pid: String, sender: ActorRef, writerUuid: String): Unit = {
     val msgs =
       if (supportsAtomicPersistAllOfSeveralEvents)
```

Comment:
I think I've just massed up with closing curly braces of `map { i => ... } `, moment :)